### PR TITLE
Curator metadata

### PR DESCRIPTION
**Pull request by daisieh**
First pass at the curator edit-metadata form with authority and confidence visible.

### CODE REVIEW

**Comment by daisieh** [comment]
Must run github.com/datadryad/dryad-utils/pull/24 on the database after deploying this.

**Comment by ryscher** [comment]
These changes only affect archived items -- the authority values cannot be seen/edited for items in curation. Is that intentional?

File: dspace/modules/api/src/main/java/org/dspace/workflow/DryadWorkflowUtils.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ public static String getAuthors(Item item) {
 
             // check for orcid:
             if (metadata.authority != null) {
-                Pattern p = Pattern.compile(".+orcid:(.+)");
+                Pattern p = Pattern.compile(".*orcid:(.+)");
```
Comment (ryscher) [inline dspace/modules/api/src/main/java/org/dspace/workflow/DryadWorkflowUtils.java]: Why is this change needed? What text would we allow before "orcid"?

Comment (ryscher) [reply dspace/modules/api/src/main/java/org/dspace/workflow/DryadWorkflowUtils.java]: (discussed in standup -- it's ok)

**Comment by daisieh** [comment]
It works for items in curation now.

**Comment by ryscher** [comment]
Looks good, tested on VM.
